scss/components/shellbar.scss

```diff
@@ -257,4 +257,44 @@ $block-user-menu: #{$fd-namespace}-user-menu;
     &__control {
       padding: fd-space(1);
     }
+}
+
+$block-product-menu: #{$fd-namespace}-product-menu;
+
+.#{$block-product-menu} {
+  $fd-product-menu-color: var(--fd-color-shell-2);
+  padding-right: 8px;
+  @include fd-rtl() {
+    padding-right: 0;
+    padding-left: 8px;
+  }
+  &__control {
+    @include fd-var-color("background-color", $fd-product-menu-color, --fd-color-shell-2);
+    position: relative;
+    @include fd-button-reset();
+    @include fd-var-size("height", $fd-forms-height, --fd-forms-height);
+    position: relative;
+    padding-right: 20px;
+    padding-left: 0;
+    @include fd-rtl() {
+      padding-right: 0;
+      padding-left: 20px;
+    }
+    &::after {
+      @include triangle(10px 5px, fd-color("shell", 2), down, --fd-color-shell-2);
+      content: "";
+      position: absolute;
+      top: calc(50% - 2.5px);
+      margin-left: 8px;
+      margin-right: 8px;
+    }
+  }
+  .fd-popover__body--right {
```

Comment:
This block is most likely fundamental-styles specific as popovers will live separately in the dom in the implementation libraries. See: https://github.com/SAP/fundamental-styles/blob/master/docs/_sass/_docs-display-component.scss#L43